### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by jodh-intel**
Quote code blocks and don't use apostrophes to quote words as this defeats spell checkers.

Also added Tables of Contents and missing entries.

Fixes: #164.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: deployment/packet/README.md [hunk 1]
```diff
@@ -155,13 +165,13 @@ $ export PATH=/usr/local/go/bin:$PATH
...
 
-## Configuring checkmetrics
+## Configuring `checkmetrics`
 
 A metrics machine must have a valid `checkmetrics` configuration file in `/etc/checkmetrics`
 in order to verify the metrics CI results.
 
 To aid setup of this file on the bare metal metrics slaves, the `add_checkmetrics.yaml`
-ansible script can be used. A local checkmetrics toml file must be present in the same
+Ansible script can be used. A local `checkmetrics` TOML file must be present in the same`
```
Comment (devimc) [inline deployment/packet/README.md]: `ansible` ?

Comment (grahamwhaley) [reply deployment/packet/README.md]: yep, as in https://www.ansible.com/ ... not sure what your intention was @devimc - just surprised, or you wanted the capitalisation turned back? :-)

Comment (grahamwhaley) [reply deployment/packet/README.md]: Oh, I see - you mean `code block` it - I see....

Comment (jodh-intel) [reply deployment/packet/README.md]: I was treating this as the "project name" (Ansible) but should it be treated as a command name then?

Comment (grahamwhaley) [reply deployment/packet/README.md]: I think in that context it is the project name.

**Pull request merged**